blazor/pivot-table/sorting.md

```diff
@@ -168,16 +168,16 @@ N> Value fields are set to the column axis by default. In such cases, the value
 ## Events
 ### OnActionBegin
 
-The event [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewEvents-1.html#Syncfusion_Blazor_PivotView_PivotViewEvents_1_OnActionBegin) triggers when clicking the value sort icon or the sort icon in the field button, which is present in both grouping bar and field list UI. This allows user to identify the current action being performed at runtime. It has the following parameters:
+The event [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewEvents-1.html#Syncfusion_Blazor_PivotView_PivotViewEvents_1_OnActionBeginvb) triggers when clicking the value sort icon or the sort icon in the field button, which is present in both grouping bar and field list UI. This allows user to identify the current action being performed at runtime. It has the following parameters:
```

Comment:
recheck the link. seems additionally you added vb in that link.
<img width="980" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/44014283/19ecd72f-8b8b-4012-932b-99710ff95ba8">

Confirm if you open your link, it goes to corresponding property page?
<img width="840" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/44014283/cd2803e6-277e-407a-9044-5688c5058630">

Reply:
Resolved